spec/views/teacher_interface/pages/eligible_spec.rb

```diff
@@ -0,0 +1,79 @@
+require "rails_helper"
+
+RSpec.describe "teacher_interface/pages/eligible", type: :view do
```

Comment:
Ah, view specs, nice. I was going to suggest that another possible improvement could be to extract the conditional content in into a [ViewComponent](https://viewcomponent.org/). This works as well. :+1: